Sources/MessagingInApp/Gist/Gist.swift

```diff
@@ -140,9 +140,9 @@ public class Gist: GistDelegate {
         if let queueId = message.queueId {
             shownMessageQueueIds.insert(queueId)
         }
-        let userToken = UserManager().getUserToken()
-        LogManager(siteId: siteId, dataCenter: dataCenter)
-            .logView(message: message, userToken: userToken) { response in
+        let state = InAppMessageState()
```

Comment:
i dont understand this one, why aren't we using the state from manager?

Reply:
This is just to keep things compiling. As I mentioned above, this class is almost completely rewritten in PR #802